notebooks/work-in-progress/explore_fuel_allocation.ipynb

```diff
@@ -2,8 +2,8 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    def remove_off_msw_fuels(df, idx):
</pre><p>Are you <strong>removing</strong> these fuels? Or just renaming them? Would it make sense to do this earlier in the process somewhere before the fuel aggregation happens? Or is the groupby().sum() just meant to consolidate what were previously distinct records into a single record? Like all the other records should be left untouched by this operation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>